FlightStreamDeck.Logics/ImageLogic.cs

```diff
@@ -217,12 +225,93 @@ public string GetGaugeImage(string text, float value, float min, float max)
                 if (!string.IsNullOrWhiteSpace(text))
                 {
                     var size = TextMeasurer.Measure(text, new RendererOptions(titleFont));
-                    ctx.DrawText(text, titleFont, Color.White, new PointF(HALF_WIDTH - size.Width / 2, 57));
+                    ctx.DrawText(text, titleFont, Color.White, new PointF(HALF_WIDTH - size.Width / 3, 46));
                 }
 
                 var valueText = value.ToString();
                 Color textColor = value > max ? Color.Red : Color.White;
-                ctx.DrawText(valueText, font, textColor, new PointF(25, 30));
+                ctx.DrawText(valueText, font, textColor, new PointF(18, 20));
+            });
+
```

Comment:
extraneous empty lines

Reply:
Agree with this. Should reserve double new line when you really want to create a "special" separation.